packages/mdc-textfield/index.js

```diff
@@ -34,7 +36,7 @@ class MDCTextField extends MDCComponent {
    */
   constructor(...args) {
     super(...args);
-    /** @private {?Element} */
+    /** @private {?MDCTextFieldInput} */
```

Comment:
you can remove the ? here too

Reply:
I think we want to replace `?` with `!` since input is required